functions/pollQueue.js

```diff
@@ -1,214 +0,0 @@
-let lastHeartbeat;
-
-module.exports.handler = async function(event, context) {
-  const constants = require("../lib/constants");
-  const { POLL_DELAY } = constants;
-
-  rateHits = [];
-  lastHeartbeat = false;
-
-  try {
-    await acquireExecutionLock(process.env, constants);
-  } catch (err) {
-    console.warn("Could not acquire execution mutex", err);
-    return;
-  }
-  console.info("Execution mutex acquired");
-
-  try {
-    await sendHeartbeatMetrics(process.env, context);
-  } catch (err) {
-    console.warn("Failed to send initial heartbeat metrics", err);
-  }
-
-  let polls = 0;
-  console.info("Poller start");
-  while (Math.floor(context.getRemainingTimeInMillis() / 1000) >= 1) {
-    try {
-      const tname = `pollQueue ${++polls}`;
-      console.time(tname);
-      await pollQueue(process.env, constants, context);
-      console.timeEnd(tname);
-    } catch (err) {
-      console.error("Error in pollQueue", err);
-      return;
-    }
-
-    try {
-      await maybeSendHeartbeatMetrics(process.env, constants, context);
-    } catch (err) {
-      console.warn("Failed to send periodic heartbeat metrics", err);
-    }
-
-    console.info("Pausing for", POLL_DELAY, "ms");
-    await wait(POLL_DELAY);
-    console.info("Remaining", context.getRemainingTimeInMillis(), "ms");
-  }
-  console.info("Poller exit");
-
-  try {
-    await releaseExecutionLock(process.env, constants);
-  } catch (err) {
-    console.warn("Could not release execution mutex", err);
-    return;
-  }
-  console.info("Execution mutex released");
-
-  try {
-    await sendHeartbeatMetrics(process.env, context);
-  } catch (err) {
-    console.warn("Failed to send final heartbeat metrics", err);
-  }
-};
-
-const wait = delay => new Promise(resolve => setTimeout(resolve, delay));
-
-const acquireExecutionLock = (
-  { CONFIG_TABLE },
-  { EXECUTION_MUTEX_KEY, EXECUTION_MUTEX_TTL }
-) =>
-  DBD.put({
-    TableName: CONFIG_TABLE,
-    Item: {
-      key: EXECUTION_MUTEX_KEY,
-      value: Date.now() + EXECUTION_MUTEX_TTL
-    },
-    ConditionExpression: "#key <> :key OR (#key = :key AND #value < :value)",
-    ExpressionAttributeNames: { "#key": "key", "#value": "value" },
-    ExpressionAttributeValues: {
-      ":key": EXECUTION_MUTEX_KEY,
-      ":value": Date.now()
-    }
-  }).promise();
-
-const releaseExecutionLock = (
-  { CONFIG_TABLE },
-  { EXECUTION_MUTEX_KEY, EXECUTION_MUTEX_TTL }
-) =>
-  DBD.delete({
-    TableName: CONFIG_TABLE,
-    Key: { key: EXECUTION_MUTEX_KEY }
-  }).promise();
-
-// Throttle sending heartbeat metrics
-const maybeSendHeartbeatMetrics = async (env, constants, context) => {
-  if (
-    lastHeartbeat !== false &&
-    Date.now() - lastHeartbeat < constants.MIN_HEARTBEAT_PERIOD
-  ) {
-    console.info("Skipping heartbeat metrics ping", Date.now() - lastHeartbeat);
-    return;
-  }
-  await sendHeartbeatMetrics(env, context);
-};
-
-const sendHeartbeatMetrics = async (
```

Comment:
Just realized that fully removing `pollQueue` also removes this periodic metrics ping with general queue health info. Creating issue #118 to re-add this as a simple function we can call every minute to send this ping.